main.go

```diff
@@ -224,12 +244,22 @@ func parseLogFlags(args logOptionsType) error {
 	return nil
 }
 
+func ShowVersion() {
+	v := fmt.Sprintf("%s\n", CreateVersionString())
+	os.Stderr.Write([]byte(v))
```

Comment:
On Fri, May 6, 2016 at 11:45 AM, Kristian Amlie <notifications@github.com>
wrote:

> In main.go
> <https://github.com/mendersoftware/mender/pull/47#discussion_r62308778>:
>
> > @@ -224,12 +244,22 @@ func parseLogFlags(args logOptionsType) error {
> >  	return nil
> >  }
> >
> > +func ShowVersion() {
> > +	v := fmt.Sprintf("%s\n", CreateVersionString())
> > +	os.Stderr.Write([]byte(v))
>
> Should this perhaps be Stdout instead? It might be easier to use in
> scripts.
>

stdout works too, I'll push another patch


-- 
Maciej Borzecki